### PR TITLE
Added phrases "lmao" and "lol" to ChatSanitizationManager

### PR DESCRIPTION
**Pull request by Venomii**
Users now simply emote laugh when they say "lmao" or "lol"

:cl:
- add: Added automatic laugh emote when saying "lol" or "lmao".

### CODE REVIEW

File: Content.Server/Chat/Managers/ChatSanitizationManager.cs [hunk 1]
```diff
@@ -59,6 +59,8 @@ public class ChatSanitizationManager : IChatSanitizationManager
         { "^^/", "chatsan-waves" },
         { ":/", "chatsan-uncertain" },
         { ":\\", "chatsan-uncertain" },
+        { "lmao", "chatsan-smiles" },
+        { "lol", "chatsan-smiles" },
```
Comment (Zumorica) [inline Content.Server/Chat/Managers/ChatSanitizationManager.cs:63]: I think `chatsan-laughs` would work better here

Comment (Venomii) [reply Content.Server/Chat/Managers/ChatSanitizationManager.cs]: Yeah actually, changing this now. that was an oversight on my part

Comment (Venomii) [reply Content.Server/Chat/Managers/ChatSanitizationManager.cs]: > I think `chatsan-laughs` would work better here

updated

**Review by Zumorica — APPROVED** [review]
LGTM, but requesting a review from @moonheart08 first before merge

**Review by PaulRitter — APPROVED** [review]

**Comment by moonheart08** [comment]
lgtm

**Comment by DogZeroX** [comment]
You forgot lmfao

**Pull request merged**